pyModeS/decoder/adsb.py

```diff
@@ -19,6 +19,7 @@
 
 from __future__ import absolute_import, print_function, division
 from pyModeS.decoder import common
+import pandas as pd # JoseAndresMR: I suppose you already have pandas in a common function, just change all occurrencies
```

Comment:
In common.py,  we only used the numpy library so far. However, it is also possible to read csv in numpy. I will check whether it worth to add the additional dependence of pandas.

Reply:
As you prefer, for me is the same. I had never used csv only with numpy.

Reply:
When a decision is taken, let me know and I will make the change.

Reply:
let's keep it as is for in this pull. I am going to merge and test with pandas first. Then I can change it to numpy later.